ComputeRawCF.py

```diff
@@ -55,58 +55,85 @@
 
 hSE = {}
 hME = {}
+hMErew = {}
 for comb, fdcomb in combs.items():
     oFile.mkdir(comb)
     oFile.cd(comb)
-
+    
     hSE[comb] = {}
     hME[comb] = {}
+    hMErew[comb] = {}
     for region in regions:
-        hSE[comb][region] = Load(inFile, f'HMResults0/HMResults0/{fdcomb}/SEDist_{fdcomb}')
-        hME[comb][region] = Load(inFile, f'HMResults0/HMResults0/{fdcomb}/MEDist_{fdcomb}')
-
+        pcsuffix = cfg['pcsuffix']
+        hSE[comb][region] = Load(inFile, f'HMResults{pcsuffix}/HMResults{pcsuffix}/{fdcomb}/SEDist_{fdcomb}')
+        hME[comb][region] = Load(inFile, f'HMResults{pcsuffix}/HMResults{pcsuffix}/{fdcomb}/MEDist_{fdcomb}')
+        hSEmultk = Load(inFile, f'HMResults{pcsuffix}/HMResults{pcsuffix}/{fdcomb}/SEMultDist_{fdcomb}')
+        hMEmultk = Load(inFile, f'HMResults{pcsuffix}/HMResults{pcsuffix}/{fdcomb}/MEMultDist_{fdcomb}')
+        nbins = hMEmultk.ProjectionX().GetNbinsX()
+        hMEreweightk = TH1F("MErewdistr", "MErewdistr", nbins, hMEmultk.GetXaxis().GetXmin(), hMEmultk.GetXaxis().GetXmax())
+        for iBin in range(hMEmultk.ProjectionY().GetNbinsX()):
+            hSEbinmult = hSEmultk.ProjectionX(f'{comb}SEdistr', iBin, iBin+1)
+            hMEbinmult = hMEmultk.ProjectionX(f'{comb}MEdistr', iBin, iBin+1)
```

Comment:
range starts from zero so if you loop over [0, nbins+2) you loop over all the bins (under- and over-flow included). Also if you project from `iBin` to `iBin+1` you are including 2 bins in the projection

```suggestion
        for iBin in range(hMEmultk.ProjectionY().GetNbinsX() + 2): # Loop over underflow, all bins, and overflow
            hSEbinmult = hSEmultk.ProjectionX(f'{comb}SEdistr', iBin, iBin)
            hMEbinmult = hMEmultk.ProjectionX(f'{comb}MEdistr', iBin, iBin)
```